app/controllers/account_details.js

```diff
@@ -25,6 +25,13 @@ export default Controller.extend({
   preferredContactNumber: "",
   mobilePhone: "",
   isMobileApp: config.cordova.enabled,
+  userInfoError: "",
+  firstName: computed("model.user.firstName", function() {
+    return $.trim(this.get("model.user.firstName"));
```

Comment:
Don't go jQuery way, use js.

```suggestion
    return this.get("model.user.firstName").trim();
```